### DISCUSSION TITLE
CVE-2023-0464 - Estimate on resolution

### DISCUSSION THREAD

**Discussion by peterbernhardt** [openssl/openssl#22125] [Q&A] [answered]
I am working on resolving any security vulnerabilities my company's repos may have with the 3rd party libraries we use and learned about this one in openssl.

Is there an estimate on when this will be resolved? I see there are [three open issues](https://github.com/openssl/openssl/issues?q=is%3Aissue+is%3Aopen+CVE-2023-0464) but the most recent update is from July with nothing pointing to a fix in the works.

When can we expect this to be resolved?

More context from our scan result:

> A security vulnerability has been identified in all supported versions
> of OpenSSL related to the verification of X.509 certificate chains
> that include policy constraints.  Attackers may be able to exploit this
> vulnerability by creating a malicious certificate chain that triggers
> exponential use of computational resources, leading to a denial-of-service
> (DoS) attack on affected systems.
> 
> Policy processing is disabled by default but can be enabled by passing
> the `-policy' argument to the command line utilities or by calling the
> `X509_VERIFY_PARAM_set1_policies()' function.

**Comment by paulidale** [comment, marked as answer]
Have a look at our [vulnerabilities page](https://www.openssl.org/news/vulnerabilities.html#CVE-2023-0464) for versions where this was fixed.
